pkg/plugin/work-in-progress/event_handler_gh_test.go

```diff
@@ -80,8 +84,31 @@ var _ = Describe("Test Keeper Plugin features", func() {
 			Ω(err).ShouldNot(HaveOccurred())
 		})
 
+		It("should mark opened PR as work-in-progress when prefixed with configured prefix", func() {
```

Comment:
`should mark opened PR as work-in-progress when title starts with configured prefix`